language/documentation/examples/experimental/verify-sort/sources/loop.move

```diff
@@ -0,0 +1,66 @@
+// exclude_for: cvc5
+module 0x42::VerifySort {
+
+    use std::vector;
+    // iperm is a ghost variable for verification
+    public fun verify_sort(v: &mut vector<u64>, iperm: &mut vector<u64>) {
```

Comment:
The prover has limited ghost variables now.  This is a use case to justify the addition of "ghost parameters".   Can we file an issue so it's not forgotten?

Reply:
New issue #216 filed.